### DISCUSSION TITLE
Custom test area field text editor

### DISCUSSION THREAD

**Discussion by acoolov** [YetiForceCompany/YetiForceCRM#15279] [How to] [answered]
Hello,

When I create a custom text area field called Equipment/Material in Yetiforce, it adds a field but the field does not have the same text editor like the Description or Solution fields have. How can I add the same field type like the Description field so it has the same text editor?
![Screenshot (7)](https://user-images.githubusercontent.com/38298689/136263056-b83e1c81-49cc-4e31-a87f-712a57059b9d.png)

**Comment by rskrzypczak** [comment, marked as answer]
Use WYSIWYG Editor field type
![image](https://user-images.githubusercontent.com/10221999/136317628-b8d5eeab-5861-472c-81fb-afd8e89474a2.png)

**Comment by acoolov** [comment]
Thanks.
